Lib/test/test_os.py

```diff
@@ -2164,7 +2159,10 @@ def helper(self):
                 self.check(getattr(os, f))
         return helper
     for f in singles:
-        locals()["test_"+f] = get_single(f)
+        if f in ("fstat", "fsync"):
+            locals()["test_"+f] = unittest.expectedFailureIfWindows("TODO: RUSTPYTHON fstat test (OSError: [Errno 18] There are no more files.")(get_single(f))
+        else:
+            locals()["test_"+f] = get_single(f)
```

Comment:
I think it may be best to add a comment before the `if` statement that explains what exactly is going on and tells future contributors what to do with this logic.

Reply:
I though about it, but in this case we can forget to remove this if, when those function will be implemented.
AFAIK `expectedFailureIfWindows` fails if test succeeds.

Reply:
It does fail, still, I agree, a comment would be nice. Something like:

```python
# TODO: RUSTPYTHON: 'fstat' and 'fsync' currently fail on windows, so we've added the if
# statement here to wrap them. When completed remove the if clause and just leave a call to:
# locals()["test_"+f] = get_single(f)
```

would do. Slightly verbose but could save someone a headache.

Reply:
done